package.json

```diff
@@ -66,7 +66,7 @@
   "bundlewatch": {
     "files": [
       {
-        "path": "dist/nearlib.min.js",
+        "path": "dist/near-api-js.min.js",
```

Comment:
I noticed the `/dist` folder still has nearlib in it. I think we should run `yarn dist` and remove those, right?
https://github.com/near/near-api-js/tree/fix-nearlib-rename/dist

Reply:
@mikedotexe this is best not done in PRs (results in merge conflicts). Will need rebuild after merge anyway.

Reply:
okay, so typically these are pushed to master, then?